requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/edx/XBlock.git#egg=XBlock==0.4.12
+git+https://github.com/edx/xblock-utils.git#egg=xblock-utils==1.0.2
+django==1.8.12
```

Comment:
No, there's actually no reason to make it this strict. The reason I put it here -- to make Travis CI install bare minimum to run tests.

I'll relax this requirement to `Django >= 1.8, < 1.9` as it is in the [XBlock's requirements](https://github.com/edx/XBlock/blob/xblock-0.4.12/django_requirements.txt)